config/development.json

```diff
@@ -44,7 +44,6 @@
 		"devdocs/redirect-loggedout-homepage": true,
 		"devdocs/components-usage-stats": false,
 		"dev/test-helper": true,
-		"dev/preferences-helper": true,
```

Comment:
Probably this line should not be touched?

Reply:
bad rebase- removed 👍